next-tavla/app/(admin)/organizations/[id]/page.tsx

```diff
@@ -50,14 +50,16 @@ async function EditOrganizationPage({ params }: TProps) {
 
     return (
         <main className={classes.root}>
-            <Heading1 margin="top">{organization.name}</Heading1>
-            <LeadParagraph margin="none" className="mb-4">
-                Valgene som tas blir satt som standard når det opprettes en
-                tavle i organisasjonen &quot;{organization.name}&quot;. Valgene
-                kan fortsatt justeres i hver enkelt tavle (med unntak av logo).
-                Innstillingene vil ikke påvirke tavler som allerede har blitt
-                opprettet.
-            </LeadParagraph>
+            <div>
```

Comment:
hva er poenget med denne diven

Reply:
det var gap mellom `heading` og `leadparagraph`, som kom av `classes.root`. La en div rundt for å fjerne den spacingen

Reply:
kan vi ikke da bare fjerne gap hvis vi ikke trenger det

Reply:
Jeg skjønner ikke helt hvorfor vi vil fjerne så mye spacing, burde det ikke være en ish lik layout over alle sider? Jeg tror ikke vi bare kan fjerne gap heller siden det er flere sider som bruker den stylingen

Reply:
Vi kan sikkert droppe å bruke root klassen her, den er jo ikke spesielt magisk uansett

Reply:
Jeg har egentlig fikset på dette her tidligere 🤔 Det bildet som er i notion-kortet er gammelt tror jeg. Det har jo mye større spacing enn det som er ditt "before" bilde her @purusott

Reply:
Kanskje bare droppe hele denne PRen?

Reply:
😭